src/kibana/components/visualize/visualize.html

```diff
@@ -2,7 +2,7 @@
   class="text-center visualize-error visualize-chart">
   <div class="item top"></div>
   <div class="item">
-    <h2><i class="fa fa-meh-o"></i></h2>
+    <h2><i aria-hidden="true" class="fa fa-meh-o"></i></h2>
```

Comment:
Font icons, such as Font-Awesome, use psuedo elements to accomplish their goals. Unfortunately, VoiceOver on OS X as well as many other assistive technologies can't read the content of pseudo elements. ([reference 1](http://www.456bereastreet.com/archive/201205/css_generated_content_and_screen_readers/), [reference 2](http://cssgallery.info/testing-the-accessibility-of-the-css-generated-content/)). For this reason we use the `aria-hidden` attribute to prevent it from being spoken.

I don't believe there is a problem specifically with the empty h2 though it would probably announce it as HTML content there. It would require one additional `aria-hidden` on the h2 if both needed to be ignored.